site/content/installation/installing-ngf/manifests.md

```diff
@@ -61,7 +61,7 @@ Deploying NGINX Gateway Fabric with Kubernetes manifests takes only a few steps.
 
 ##### For NGINX Plus
 
-   Download the [deployment YAML](https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.2.0/nginx-plus-gateway.yaml).
+   Download the [deployment YAML](https://github.com/nginxinc/nginx-gateway-fabric/releases/download/v1.1.0/nginx-plus-gateway.yaml).
```

Comment:
This was purposely set to v1.2.0 because this YAML doesn't exist yet. This part of the doc isn't published yet, so users won't see it.

Reply:
yep just aligning all the tags so everything gets updated at the same time